config/base/default-clusterroles.yaml

```diff
@@ -16,6 +16,9 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: readonly
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
```

Comment:
Why do we need `aggregate-to-edit` here? Shouldn't that be added to the `readwrite` ClusterRole?

Reply:
OK, that explains why not to add `aggregate-to-edit` on the `readwrite` ClusterRole. But it does not explain why add it to the `readonly` ClusterRole?

Reply:
Because it should have at least have view role.

Reply:
But it has. On the next line.

Reply:
It is for the view accounts. I do not think that affects the the edit accounts.
Let me repeat my original question though. Why do we need both `aggregate-to-edit` and `aggregate-to-view` labels on the `readonly` role? Isn't the latter enough?